src/Stryker.Core/Stryker.Core/TestRunners/VsTest/VsTestRunner.cs

```diff
@@ -230,7 +258,7 @@ private void ParseResultsForCoverage(IEnumerable<TestResult> testResults)
                     var propertyPairValue = (value as string);
                     if (string.IsNullOrWhiteSpace(propertyPairValue))
                     {
-                        CoverageMutants.DeclareMappingForATest(testResult.TestCase, new List<int>(), new List<int>());
+                        //
```

Comment:
Is there supposed to be nothing here>

Reply:
Added log to clarify what is (not) happening here